### ISSUE TITLE
Warning: losoto multithread crash leaves pipeline hanging

### ISSUE THREAD

**Issue by ajstewart** [lofar-astron/PiLL#2]
This is a warning rather than an issue as I believe it needs to be corrected at the losoto end. If losoto crashes during a multithread operation the pipeline environment does not get a crash signal and just proceeds to hang. Indeed testing losoto individually also just hangs when such an error occurs.

I experienced this while running the pipeline with matplotlib v2.0.0 installed - https://github.com/revoltek/losoto/issues/16.

This may be already known and can be closed if so, but I just wanted to record it somewhere.

**Comment by adrabent** [comment]
This issue occured already and was thought to be fixed already?
Does this issue re-occur after updaing to matplotlib v2.0.0?

**Comment by ajstewart** [comment]
Yes losoto could not generate the plots because of some matplotlib file locks in v2.0.0, and the pipeline was left hanging. To get PiLL/losoto to get past this I had to downgrade to matplotlib 1.5.2 which works.

In the issue I referenced above, I reference a matplotlib issue that seems to be the cause, but I'm unclear if it's by design or an issue in matplotlib v2.0.0 that will be fixed.
